saleor/graphql/shipping/utils.py

```diff
@@ -53,13 +47,7 @@ def annotate_shipping_methods_with_price(
     }
     for method in shipping_methods:
         shipping_channel_listing = channel_listing_map[method.id]
-        taxed_price = manager.apply_taxes_to_shipping(
```

Comment:
Blessed in SALEOR-4842 due to performance / inconsistency. `Checkout.shippingMethod` was always untaxed, while `Checkout.availableShippingMethods` was taxed